airflow/providers/microsoft/azure/operators/azure_service_bus_queue.py

```diff
@@ -0,0 +1,182 @@
+        self.dead_lettering_on_message_expiration = dead_lettering_on_message_expiration
+        self.enable_batched_operations = enable_batched_operations
+        self.azure_service_bus_conn_id = azure_service_bus_conn_id
+
+    def execute(self, context: "Context") -> None:
+        """Creates Queue in Azure Service Bus namespace, by connecting to Service Bus Admin client in hook"""
+        hook = AzureServiceBusAdminClientHook(azure_service_bus_conn_id=self.azure_service_bus_conn_id)
+
+        # create queue with name
+        queue = hook.create_queue(
+            self.queue_name,
+            self.max_delivery_count,
+            self.dead_lettering_on_message_expiration,
+            self.enable_batched_operations,
+        )
+        self.log.info("Created Queue ", queue)
+
+
+class AzureServiceBusSendMessageOperator(BaseOperator):
+    """
+    Send Message or batch message to the service bus queue
+
+    :param message: Message which needs to be sent to the queue. It can be string or list of string.
+    :param batch: Its boolean flag by default it is set to False, if the message needs to be sent
+        as batch message it can be set to True.
+    :param azure_service_bus_conn_id: Reference to the
+        :ref: `Azure Service Bus connection<howto/connection:azure_service_bus>`.
+    """
+
+    template_fields: Sequence[str] = ("queue_name",)
+    ui_color = "#e4f0e8"
+
+    def __init__(
+        self,
+        *,
+        queue_name: str,
+        message: Union[str, List[str]],
+        batch: bool = False,
+        azure_service_bus_conn_id: str = 'azure_service_bus_default',
+        **kwargs,
+    ) -> None:
+        super().__init__(**kwargs)
+        self.queue_name = queue_name
+        self.batch = batch
+        self.message = message
+        self.azure_service_bus_conn_id = azure_service_bus_conn_id
+
+    def execute(self, context: "Context") -> None:
+        """
+        Sends Message to the specific queue in Service Bus namespace, by
+        connecting to Service Bus  client
+        """
+        # Create the hook
+        hook = ServiceBusMessageHook(azure_service_bus_conn_id=self.azure_service_bus_conn_id)
+
+        # send message
+        hook.send_message(self.queue_name, self.message, self.batch)
+
+
+class AzureServiceBusReceiveMessageOperator(BaseOperator):
+    """
+    Receive a batch of messages at once in a specified Queue name
+
+    :param queue_name: The name of the queue in Service Bus namespace.
+    :param azure_service_bus_conn_id: Reference to the
+        :ref: `Azure Service Bus connection <howto/connection:azure_service_bus>`.
+    """
+
+    template_fields: Sequence[str] = ("queue_name",)
+    ui_color = "#e4f0e8"
+
+    def __init__(
+        self,
+        *,
+        queue_name: str,
+        azure_service_bus_conn_id: str = 'azure_service_bus_default',
+        max_message_count: Optional[int] = 10,
+        max_wait_time: Optional[float] = 5,
```

Comment:
```suggestion
        max_message_count: int = 10,
        max_wait_time: float = 5,
```
Can you add these parameters to the operator's docstring too?

Reply:
Added params to doc string